domain-models-runtime/src/main/java/org/folio/rest/impl/TenantAPI.java

```diff
@@ -123,7 +105,7 @@ String getTablePath() {
    * @throws NoSchemaJsonException when templates/db_scripts/schema.json doesn't exist
    * @throws TemplateException when processing templates/db_scripts/schema.json fails
    */
-  public String sqlFile(String tenantId, boolean tenantExists, TenantAttributes tenantAttributes,
+  public String [] sqlFile(String tenantId, boolean tenantExists, TenantAttributes tenantAttributes,
```

Comment:
Javadoc should explain the content of the returned array.

Reply:
done